command/ca/provisionerbeta/list.go

```diff
@@ -0,0 +1,68 @@
+package provisionerbeta
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/certificates/pki"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/flags"
+	"github.com/urfave/cli"
+)
+
+// nolint
+func listCommand() cli.Command {
+	return cli.Command{
+		Name:   "list",
+		Action: cli.ActionFunc(listAction),
+		Usage:  "list provisioners configured in the CA",
+		UsageText: `**step ca provisioner list**
```

Comment:
beta is missing

Reply:
These methods aren't used. But I added beta anyway.